src/mlpack/tests/CMakeLists.txt

```diff
@@ -163,12 +157,16 @@ add_executable(mlpack_catch_test
   main_tests/knn_test.cpp
   main_tests/krann_test.cpp
   main_tests/linear_regression_test.cpp
-  main_tests/lmnn_test.cpp
```

Comment:
not sure about this change

Reply:
this was a mistake during the merge conflict fix. fixed it.